toolchain/lower/mangler.cpp

```diff
@@ -210,14 +210,11 @@ auto Mangler::MangleGlobalVariable(SemIR::InstId pattern_id) -> std::string {
 }
 
 auto Mangler::MangleCppClang(const clang::NamedDecl* decl) -> std::string {
+  auto* cpp_code_generator = file_context_.cpp_code_generator();
   CARBON_CHECK(
-      cpp_mangle_context_,
-      "Mangling of a C++ imported declaration without a Clang `MangleContext`");
-
-  RawStringOstream cpp_mangled_name;
-  cpp_mangle_context_->mangleName(decl, cpp_mangled_name);
-
-  return cpp_mangled_name.TakeStr();
+      cpp_code_generator,
+      "Mangling of a C++ imported declaration without a Clang `CodeGenerator`");
```

Comment:
Perhaps consider putting this inside cpp_code_generator() and having it return `&` instead?

Reply:
Done.